lib/kitchen/verifier/inspec.rb

```diff
@@ -36,12 +36,24 @@ class Inspec < Kitchen::Verifier::Base
 
       default_config :inspec_tests, []
 
+      def add_target(runner, target, opts)
```

Comment:
@vjeffrey @arlimus I am not sure why we are re-implementing https://github.com/chef/inspec/blob/1f5185638669136d962e0550aad3281c0d4d7630/lib/inspec/runner.rb#L49 here